services/web/server/src/simcore_service_webserver/login/decorators.py

```diff
@@ -17,7 +15,8 @@ def user_to_request(handler):
     @wraps(handler)
     async def wrapped(*args, **kwargs):
         request = get_request(*args, **kwargs)
-        request[RQT_USERID_KEY] = await authorized_userid(request)
+        userid = await check_authorized(request)
```

Comment:
you have changed the behaviour here with your change. Now it raises an exception while it was suppose to be a safe decorator. There was no need to change this for the purpose of the PR. On the other hand it was not documented so it is a fair mistake and the function is not used anymore. Just remove the entire function

Reply:
```
def login_required(handler):
    """Decorator that restrict access only for authorized users.

    User is considered authorized if authorized_userid
    returns some value.

    Keeps userid in request[RQT_USERID_KEY]
    """

    @wraps(handler)
    async def wrapped(*args, **kwargs):
        request = get_request(*args, **kwargs)
        userid = await authorized_userid(request)
        if userid is None:
            raise web.HTTPUnauthorized

        request[RQT_USERID_KEY] = userid
        ret = await handler(*args, **kwargs)
        return ret

    return wrapped
```
this is the code in master, it raises the same exception than ``check_authorized``  does

Reply:
but what do you mean remove the entire entire function? do you want me to remove the use of the decorator in the whole repo? I can do that...

Reply:
please read carefully. I am commenting the body of ``user_to_request`` and NOT ``login_required``.

Reply:
done